WordPress/Jetpack/Resources/release_notes.txt

```diff
@@ -1,7 +1,6 @@
-We made some performance improvements to the Gallery and Image blocks in the Block Editor. This will help images take up less storage on your site and keep your device from slowing down during media uploads. We feel the need for speed.
-
-If your site uses VideoPress, any Video blocks saved on the web version of the site will now display properly in the mobile app, too.
-
-The block inserter in the Block Editor now allows block types to be better organized in collections. We also made a small spacing adjustment in the alt text footnote in Image blocks, because little things are important, too.
-
-Last but not least, we squashed a pesky bug in the site creation process. When you’re horizontally scrolling through site design options, the app now remembers where you left off, even if you scroll up and down on the page.
+* [*] [Jetpack-only] Block Editor: Makes some small changes to the editor's accent colours for consistency. [#19113]
```

Comment:
total nitpick / personal take: to emphasize that I've checked the WP vs JP release note manually and that I've noted that this is Jetpack-only and removed it from the WordPress notes, as well as not make Grace wonder what this is about (i.e. wonder if this is about Jetpack-the-plugin and not Jetpack-the-app like it actually is), I usually remove the prefix from the Jetpack-specific release notes file at that point 🙃 

```suggestion
* [*] Block Editor: Makes some small changes to the editor's accent colours for consistency. [#19113]
```

OTOH I can see value in keeping this here especially to make it visible in the PR diff, so that reviewers like me think about checking that you indeed removed that entry from the WP release notes file (which you did) 🤔 🤷